opm/parser/eclipse/EclipseState/Tables/SimpleTable.hpp

```diff
@@ -60,11 +60,15 @@ namespace Opm {
          */
         double evaluate(const std::string& columnName, double xPos) const;
 
+        /// throws std::invalid_argument if jf != m_jfunc
+        void assertJFuncPressure(const bool jf) const;
```

Comment:
I just noticed this now while trying to adapt opm-material: I'm pretty sure that this method is specific to (some of) the saturation function tables, i.e., it may be a bit more convenient from an implementor's point of view, but it does not make too much sense IMO. do you mind moving this to `SwofTable` and friends?